rollup/internal/utils/exchange_rate.go

```diff
@@ -0,0 +1,73 @@
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/scroll-tech/go-ethereum/log"
+)
+
+var BinanceApiEndpoint string = "https://api.binance.com/api/v3/ticker/price?symbol=%s"
```

Comment:
Could this URL template changed later? if so, can move it to config, if not frequent, can keep it here.